components/automate-chef-io/content/docs/iam-v1-overview.md

```diff
@@ -10,14 +12,18 @@ toc = true
 [menu]
   [menu.docs]
     parent = "authorization"
-    weight = 10
+    weight = 40
 +++
 
-This guide helps you understand and use Chef Automate's IAM v1 authorization system.
+{{< warning >}}
+IAM v1 is deprecated and we recommend updating to the latest version of Automate to make use of [IAM v2]({{< relref "iam-v2-overview" >}}).
+{{< /warning >}}
+
+This guide helps you understand and use Chef Automate's IAM v1 authorization system. 
 
 ## Overview
 
-Chef Automate uses a policy-based authorization system.
+Chef Automate IAM v1 uses a policy-based authorization system.
```

Comment:
second paragraph of the Overview renders weird for me locally: 
<img width="896" alt="Screen Shot 2020-01-30 at 2 00 56 PM" src="https://user-images.githubusercontent.com/21015366/73494266-0d2e8100-4369-11ea-89c9-0824ffb0528f.png">

Reply:
same thing down in Action under the Structure section:
<img width="879" alt="Screen Shot 2020-01-30 at 2 04 18 PM" src="https://user-images.githubusercontent.com/21015366/73494493-76ae8f80-4369-11ea-95fe-83e5ea470adf.png">

Reply:
very strange, im not seeing this. but it seems to be an issue in the live version as well.

Reply:
I can _why_ its happening i think, but i'm not catching the source.  It's not pulling in the tick mark after "user@example.com" so its creating code out of the _opposites_

Reply:
oh wait i think I see it...there is another tickmark not getting caught after "compliance"

Reply:
sorry If i'm stating the obvious here - I don't have a branch pulled down right now to test it out yet 🤷‍♂

Reply:
@bcmdarroch can you run this again locally, and let me know if you still see the issue?

Reply:
This seems to be resolved...